src/elements/Root.ts

```diff
@@ -0,0 +1,10 @@
+import { styled } from '@material-ui/core';
+
+const Root = styled('div')(() => {
+  return {
+    // display: 'flex',
```

Comment:
code comment.  what is a root element?  I feel as if you may have broken this down into too small of a chunk